### DISCUSSION TITLE
Conflict Between Folder Templates and Template File

### DISCUSSION THREAD

**Discussion by macdrifter** [SilentVoid13/Templater#914] [Help]
I have been using the below template script for many months and it works (mostly) great:

```
<%*
let qcFileName = await tp.system.prompt("Note Title")
titleName = qcFileName + " " + tp.date.now("YYYY-MM-DD")
await tp.file.rename(titleName)
await tp.file.move("/Notes/" + titleName);
-%>
---
title: <% qcFileName %>
date: <% tp.file.creation_date("YYYY-MM-DD HH:mm:ss") %>
tags:
topic: 

---

<% tp.file.cursor(0) %>
```
The template asks for the note title and then creates the note in a folder called "**/Notes/**". Normally I trigger this template with the command palette but when I activated "**Enable Folder Notes**" option in Templater I am prompted twice for the "**Note Title**" property. The template is triggered once by the Templater command palette and then again when the file is moved to the "**/Notes/**" folder.

Is there a way to tell Templater that if a file is created through the Templater command palette then it should ignore the Folder Template setting?

**Comment by Zachatoo** [comment]
I usually handle this by checking if the title is "Untitled", then I know whether or not I need to get the note title.
```js
<%*
const hasTitle = !tp.file.title.startsWith("Untitled");
let title;
if (!hasTitle) {
  title = await tp.system.prompt("Title"); // if there's not already a note title, prompt for one
  await tp.file.rename(title);
} else {
  title = tp.file.title; // if there's already a note title, use that
}
_%>
# <% title %>
```

**Reply by macdrifter** [reply]
Thank you. That's a good workaround.
